File: clients/java/src/main/java/com/priostack/acn/StaleBaseError.java
package com.priostack.acn;

import com.fasterxml.jackson.databind.JsonNode;

/** {@code stale-base} — the operation raced a newer state; re-read and retry. */
public class StaleBaseError extends ACNToolError {

    private static final long serialVersionUID = 1L;

    public StaleBaseError(String outcome, String detail, String method, JsonNode data) {
        super(outcome, detail, method, data);
    }
}

File: clients/php/src/Exception/StaleBaseError.php
<?php

declare(strict_types=1);

namespace Priostack\Acn\Exception;

/** {@code stale-base} — the operation raced a newer state; re-read and retry. */
class StaleBaseError extends AcnToolError
{
    public const OUTCOME = 'stale-base';
}
